java/core/src/main/java/co/worklytics/psoxy/rules/asana/PrebuiltSanitizerRules.java

```diff
@@ -37,7 +37,8 @@ public class PrebuiltSanitizerRules {
             "modified_at.after",
             "modified_at.before",
             "is_subtask",
-            "sort_ascending"
+            "sort_ascending",
```

Comment:
From https://github.com/Worklytics/psoxy/pull/214 (merged to main, instead of RC)